lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -29,7 +29,7 @@
 
 namespace Org.Apache.REEF.Common.Runtime.Evaluator.Context
 {
-    public class ContextRuntime
+    internal sealed class ContextRuntime
```

Comment:
Do we need to go through a deprecation release? I'm in favor not to, but we should ask on the dev list.